### PR TITLE
add automation bridge with podman version

### PR DESCRIPTION
**Pull request by ffelici-sumo**
## Purpose of this pull request

This pull request (PR). Adds the documentation to configure and use the automation bridge with Podman.

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/automation-service/automation-service-bridge.md [hunk 1]
```diff
@@ -185,3 +185,48 @@ If you are using CyberArk, you must add the following certificates provided by C
 * `RootCA_new.crt`
 * `client_new.crt`
 * `client_new.pem`
+
+### Configuring automation bridge with Podman
+
+#### Enable Podman socket
+
+- Run following commands:
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-bridge.md:193]:
```suggestion
1. Run the following commands:
```

File: docs/cse/automation-service/automation-service-bridge.md [hunk 2]
```diff
@@ -185,3 +185,48 @@ If you are using CyberArk, you must add the following certificates provided by C
...
+
+```bash
+systemctl enable podman.socket && systemctl start podman.socket
+```
+
+- Create a symbolic link:
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-bridge.md:199]:
```suggestion
1. Create a symbolic link:
```

File: docs/cse/automation-service/automation-service-bridge.md [hunk 3]
```diff
@@ -185,3 +185,48 @@ If you are using CyberArk, you must add the following certificates provided by C
...
+
+```bash
+ln -s /run/podman/podman.sock /var/run/docker.sock
+```
+
+#### Change automation bridge configuration
+
+- Change automation bridge configuration file /usr/lib/systemd/system/automation-bridge-worker@.service
```
Comment (jpipkin1) [inline docs/cse/automation-service/automation-service-bridge.md:207]:
```suggestion
Change the automation bridge configuration file `/usr/lib/systemd/system/automation-bridge-worker@.service`.
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@ffelici-sumo, this needs just a few minor updates.

File: docs/cse/automation-service/automation-service-bridge.md [hunk 4]
```diff
@@ -185,3 +185,48 @@ If you are using CyberArk, you must add the following certificates provided by C
...
+
+[Service]
+User=root
+EnvironmentFile=/etc/opt/automation-bridge/automation-bridge.conf
+ExecStart=/opt/automation-bridge/bin/automation-bridge -f /opt/automation-bridge/etc/user-configuration.conf -n %H-%i
+ExecStop=/bin/kill -s TERM  $MAINPID
+Restart=on-failure
+TimeoutStartSec=10
+RestartSec=10
+##
```
Comment (kimsauce) [inline docs/cse/automation-service/automation-service-bridge.md:223]: is this line with the ## intentional?

File: docs/cse/automation-service/automation-service-bridge.md [hunk 5]
```diff
@@ -185,3 +185,48 @@ If you are using CyberArk, you must add the following certificates provided by C
...
+
+```bash
+ln -s /run/podman/podman.sock /var/run/docker.sock
+```
+
+#### Change automation bridge configuration
+
+- Change automation bridge configuration file /usr/lib/systemd/system/automation-bridge-worker@.service
+
+```systemd
```
Comment (kimsauce) [inline docs/cse/automation-service/automation-service-bridge.md:209]:
```suggestion
```bash title="systemd"
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
A couple of suggestions

File: docs/cse/automation-service/automation-service-bridge.md [hunk 6]
```diff
@@ -185,3 +185,48 @@ If you are using CyberArk, you must add the following certificates provided by C
 * `RootCA_new.crt`
 * `client_new.crt`
 * `client_new.pem`
+
+### Configuring automation bridge with Podman
```
[the remark this answers is not in the document]
Comment (ffelici-sumo) [reply docs/cse/automation-service/automation-service-bridge.md:189]: They confirmed that it should be replicated on [Cloud SOAR automation bridge documentation](https://help.sumologic.com/docs/cloud-soar/cloud-soar-bridge/)

File: docs/cse/automation-service/automation-service-bridge.md (hunk 4 above)
Comment (ffelici-sumo) [reply docs/cse/automation-service/automation-service-bridge.md:223]: removed 👍🏻

File: docs/cse/automation-service/automation-service-bridge.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (ffelici-sumo) [reply docs/cse/automation-service/automation-service-bridge.md:189]: committed 👍🏻

[the remark this answers is not in the document]
Comment (jpipkin1) [reply docs/cse/automation-service/automation-service-bridge.md:189]: Thanks for adding the documentation to Cloud SOAR, @ffelici-sumo.

**Review by jpipkin1 — APPROVED** [review]
PR approved, @ffelici-sumo. Note I made some minor updates with commit "[Small layout adjustments](https://github.com/SumoLogic/sumologic-documentation/pull/3211/commits/a09a6fbb8c886478714c09173a545d443ee0bc82)".

**Comment by jpipkin1** [comment]
@kimsauce, before you approve this PR and we merge it, do you want to do any checks on it to ensure it won't cause problems with our search?

**Comment by kimsauce** [comment]
> @kimsauce, before you approve this PR and we merge it, do you want to do any checks on it to ensure it won't cause problems with our search?

Good call. I'll do it now.

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
@jpipkin1 just did a fresh `yarn install` and test on my end. merging now.

**Pull request merged**